### ISSUE TITLE
The checks in Failure technique F85 for Focus Order need to be tweaked

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3223]
The checks in https://www.w3.org/TR/WCAG20-TECHS/F85.html fail to account for the situation where the trigger has been removed.

As well, the test is quite prescriptive (go backwards).

These came out during discussion on https://github.com/w3c/wcag/pull/3214/ in the [prior survey ](https://www.w3.org/2002/09/wbs/35422/wcag2x-backlog1/) and meeting. There are some other scenarios which should be considered in this failure technique to avoid unintentionally failing techniques which are acceptable in certain circumstances

**Comment by detlevhfischer** [comment]
@mbgower It would help to have a typical example where the trigger opens a modal and is no longer present afterwards. Would that be, for example, a dialog used to confirm the deletion of the very trigger that called it up (say, date a filter category, or some teaser tile)? Or are there better / more common examples?
I was wondering what the best practice would be even if the Failure test might not prescribe a location to set the focus to. My gut feeling tells me the element immediately above the (now deleted) trigger - and failing that, the start of the page.

Feedback would help if I take a stab at rewriting F85.

**Comment by mbgower** [comment]
Here's what I wrote in that survey
> In a situation where a button exists in a table row to delete that table row, when the user activates it, the focus cannot be put back on the target. At IBM we provided guidance to put the focus on the prior UIC in the focus order. This has seemed to work well in such a situation. It matches the guidance on focus order in the dialog, but contradicts the order "forward" in the change. I'd like to see it made a bit less prescriptive.

Hope that helps!

**Comment by mbgower** [comment]
@detlevhfischer did you ever have a chance to tackle this one?

**Comment by detlevhfischer** [comment]
@mbgower Now, finally, since you kindly reminded me...

https://github.com/w3c/wcag/pull/4769

**Comment by patrickhlauke** [comment]
Closed by https://github.com/w3c/wcag/pull/4769
